snippets/language-html.cson

```diff
@@ -1,4 +1,4 @@
-'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin':
+'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin, .source.gfm':
```

Comment:
I don't necessarily agree with this change as GFM does not support most of these.